howfairis/workarounds/remove_comments_rst.py

```diff
@@ -0,0 +1,6 @@
+import warnings
+
+
+def remove_comments_rst(text, fname):
+    warnings.warn("Using workaround to remove comments from {0}.".format(fname))
```

Comment:
Lets use the proper warning so #72 can be closed
```suggestion
    warnings.warn("Unable to ignore comments in RestructuredText format of {0}, checks will also see comments".format(fname))
```

Reply:
Good with me